### PR TITLE
Use CSS in the UI #3

### LINKED ISSUE #3: Use CSS in the UI
The current one doesn't use any CSS styling.

### PR DESCRIPTION
**Pull request by yl-coder**
Fixes #3
image

### CODE REVIEW

**Comment by damithc** [comment]
How does the new GUI look?

File: src/seedu/addressbook/ui/DarkTheme.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    -fx-font-size: 12pt;
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-text-fill: white;
+    -fx-control-inner-background: derive(#1d1d1d,20%);
+}
+
+.text-area {
+    -fx-background-color: black;
+    -fx-control-inner-background: black;
+    -fx-font-size: 12px;
```
Comment (damithc) [inline src/seedu/addressbook/ui/DarkTheme.css]: Why use `pt` above but `px` here? In the screenshot, the two text sizes seem different.

**Comment by damithc** [comment]
@yl-coder is this one done? Merge if yes.

**Comment by yl-coder** [comment]
px is appear differently on different screen. So i guess i will change to pt for consistency.

**Pull request merged**